test/indice.Edi.Tests/EdiTextReaderTests.cs

```diff
@@ -1022,5 +1022,34 @@ public void EdiFact_01_Test()
             Assert.Equal("And this is the seccond", interchange.Msg.InteractiveFreeTexts[1].ToString());
             Assert.Equal("while this should come third", interchange.Msg.InteractiveFreeTexts[2].ToString());
         }
+
+        [Fact]
```

Comment:
I would like to add a "skip" explanation on the attribute `Fact`. Since we are not providing with a solution yet, this will allow me to accept the pull request without breaking the build